plugins/scaffolder/src/components/ActionsPage/ActionsPage.tsx

```diff
@@ -251,7 +251,9 @@ export const ActionsPage = () => {
 
     return (
       <>
-        <Typography variant="h6">{name}</Typography>
+        <Typography variant="h6" component="h4">
```

Comment:
Hehe, mildly confusing. But simpler than re-styling it all.

Reply:
Agree, re-read the documentation probably 3 times